apps/angular/src/app/features/anime-filter-form/anime-filter-form.component.ts

```diff
@@ -0,0 +1,99 @@
+import { Component, EventEmitter, Input, Output, ChangeDetectionStrategy, OnInit, inject, DestroyRef } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatIconModule } from '@angular/material/icon';
+import { MatButtonModule } from '@angular/material/button';
+import { ReactiveFormsModule, FormGroup } from '@angular/forms';
+import { AnimeType } from '@js-camp/core/models/anime-type';
+import { MatSelectModule } from '@angular/material/select';
+import { AnimeFilters } from '@js-camp/core/models/anime-filters';
+
+import { throttleTime } from 'rxjs';
+
+import { AnimeFilterForm, AnimeFilterFormService } from './anime-filter-form.service';
+
+/**
+ * Filter form component.
+ * Contains controls for choosing the type of anime and for searching by anime title.
+ */
+@Component({
+	selector: 'camp-anime-filter-form',
+	standalone: true,
+	templateUrl: './anime-filter-form.component.html',
+	styleUrl: './anime-filter-form.component.css',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	imports: [
+		ReactiveFormsModule,
+		MatFormFieldModule,
+		MatInputModule,
+		MatButtonModule,
+		MatIconModule,
+		MatSelectModule,
+	],
+})
+export class AnimeFilterFormComponent implements OnInit {
+
+	/** Initial value for search control. */
+	@Input({ required: true })
+	public searchValue = '';
+
+	/** Initial value for anime types control. */
+	@Input({ required: true })
+	public typesValue: AnimeType[] = [];
+
+	/** Event of anime types or search term changes. */
+	@Output()
+	public animeFiltersEvent = new EventEmitter<AnimeFilters>();
```

Comment:
You can keep outputs read-only because we're not changing it at any given point in time
```suggestion
	public readonly animeFiltersEvent = new EventEmitter<AnimeFilters>();
```